specs/deneb/p2p-interface.md

```diff
@@ -41,6 +42,22 @@ The specification of these changes continues in the same format as the network s
 
 ## Modifications in Deneb
 
+### Constant
+
+*[New in Deneb:EIP4844]*
+
+| Name                                     | Value                             | Description                                                         |
+|------------------------------------------|-----------------------------------|---------------------------------------------------------------------|
+| `BLOB_KZG_COMMITMENTS_GINDEX`            | `get_generalized_index(BeaconBlockBody, 'blob_kzg_commitments')` (= 27) | `blob_kzg_commitments` field generalized index on `BeaconBlockBody` container |
```

Comment:
I can’t find this constant to be used anywhere in the code except for the comments, is it required?

Reply:
It is used to define the `KZG_COMMITMENT_INCLUSION_PROOF_DEPTH` value.